features/cloud_pro_clone.feature

```diff
@@ -13,10 +13,10 @@ Feature: Creating golden images based on Cloud Ubuntu Pro instances
         log_file: /var/log/ubuntu-advantage.log
         """
         When I run `pro auto-attach` with sudo
-        And I run `pro status --format yaml` with sudo
+        And I run `pro api u.pro.status.is_attached.v1` as non-root
         Then stdout matches regexp:
         """
-        attached: true
+        \"is_attached\": true
```

Comment:
This comment is very nitpicky so feel free to ignore it :upside_down_face: 

I think it would be just slightly nicer to use a substring check for this instead of a regexp:
```
        Then stdout contains substring:
        """
        "is_attached": true
```

Or if we want to be more ambitious, we could write a step: "Then the machine is attached" that does the API call under-the-hood and actually parses the json and checks the value is true.

Reply:
I like the idea of the new step, as it will be easier to reuse. I will implement it